### ISSUE TITLE
Specifying dependencies only once

### ISSUE THREAD

**Issue by JavierSegoviaCordoba** [kotest/kotest#1638]
Is it planned to unify dependency artifacts like JetBrains is doing with kotlinx?

https://kotlinlang.org/docs/reference/whatsnew14.html#specifying-dependencies-only-once

**Comment by sksamuel** [comment]
Yes sir, if you check out master, I've already done a bunch.

**Comment by sksamuel** [comment]
This should be done now. If you spot anything I missed let me know.

**Comment by JavierSegoviaCordoba** [comment]
@sksamuel are docs updated? https://github.com/kotest/kotest/blob/master/doc/reference.md

**Comment by sksamuel** [comment]
Updated with what ?
